lib/src/node/compile.dart

```diff
@@ -236,42 +230,35 @@ List<AsyncCallable> _parseFunctions(Object? functions, {bool asynch = false}) {
 
   var result = <AsyncCallable>[];
   jsForEach(functions, (signature, callback) {
-    Tuple2<String, ArgumentDeclaration> tuple;
-    try {
-      tuple = ScssParser(signature).parseSignature();
-    } on SassFormatException catch (error, stackTrace) {
-      throwWithTrace(
-          SassFormatException(
-              'Invalid signature "$signature": ${error.message}', error.span),
-          stackTrace);
-    }
-
     if (!asynch) {
-      result.add(BuiltInCallable.parsed(tuple.item1, tuple.item2, (arguments) {
+      Callable? callable;
```

Comment:
```suggestion
      late Callable callable;
```

This will allow you to declare the value lazily without needing explicit null assertions (instead, the language bakes in "this was assigned" assertions at each access point).